app/konnect/analytics/reference.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Reports Reference
+content_type: reference
+badge: plus
+---
+
+## Metrics
```

Comment:
Table needs some updates to use camel case for metric names. It should be:

- Request Count
- Request per Minute
- Response Latency
- Request Size
- Response Size